src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER_OF_BALL = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        String colorOfBall = colorSupplier.getRandomColor();
+        int numberOfBall = random.nextInt(MAX_NUMBER_OF_BALL + 1);
```

Comment:
```suggestion
        int numberOfBall = random.nextInt(MAX_NUMBER_OF_BALL);
```

why do we need this?